systemtests/util_test.go

```diff
@@ -22,6 +22,44 @@ var startedContainers struct {
 	names map[string]struct{}
 }
 
+var (
+	volumeListMutex = sync.Mutex{}
+	volumeList      = map[string]struct{}{}
+)
+
+func volumeParts(volume string) (string, string) {
+	parts := strings.SplitN(volume, "/", 2)
+	return parts[0], parts[1] // panic, schmanick
+}
+
+func fqVolume(policy, volume string) string {
+	return strings.Join([]string{policy, volume}, "/")
+}
+
+func genRandomVolume() string {
```

Comment:
Could this run into collisions because genRandomString doesn't avoid collisions?